tests/components/azure_data_explorer/conftest.py

```diff
@@ -0,0 +1,142 @@
+"""Test fixtures for ADX."""
+from collections.abc import Generator
+from dataclasses import dataclass
+from datetime import timedelta
+import logging
+from typing import Any
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components.azure_data_explorer.const import (
+    CONF_FILTER,
+    CONF_SEND_INTERVAL,
+    DOMAIN,
+)
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import STATE_ON
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+from homeassistant.util.dt import utcnow
+
+from .const import (
+    AZURE_DATA_EXPLORER_PATH,
+    BASE_CONFIG_FREE,
+    BASE_CONFIG_FULL,
+    BASIC_OPTIONS,
+)
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# fixtures for both init and config flow tests
+@dataclass
+class FilterTest:
```

Comment:
This class should not be in this file, as it is not used by any fixture. Please move it to `__init__.py`